tutorials/fundamentals/compare-without-time.md

```diff
@@ -0,0 +1,30 @@
+If you want to compare two dates in JavaScript without using the time aspect, you should use
+the `toDateString()` method. It returns the date portion of the `Date` object as a string. From there,
+you can compare the two strings:
+
+```javascript
+const date1 = new Date().toDateString();
+const date2 = new Date(2000, 6, 25, 14, 39, 7).toDateString();
```

Comment:
I'm not a fan of using the `new Date(year, month, day)` constructor for dates. Let's do `new Date('2000-06-25')` instead to be more consistent with our other tutorials